assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -149,6 +150,25 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 					/>
 				) }
 			</PanelBody>
+			<PanelBody
+				title={ __(
+					'Account options',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<ToggleControl
+					label={ __(
+						'Allow shopper to sign up for a user account during checkout',
+						'woo-gutenberg-products-block'
+					) }
+					checked={ allowCreateAccount }
+					onChange={ () =>
+						setAttributes( {
+							allowCreateAccount: ! allowCreateAccount,
+						} )
+					}
+				/>
+			</PanelBody>
```

Comment:
Not an issue with this specific PR, but I found it quite confusing that if you have `Allow customers to place orders without an account` unchecked in the settings, this toggle doesn't produce any visual difference in the editor and in the frontend it changes whether the form is visible to guests or not.

That's seems quite counter-intuitive with the text associated with the toggle (`Allow shopper to sign up for a user account during checkout`). I find this very confusing in WC Core as well, but wondering if we should take the chance to try to make the text a bit more clear.